news/1129.bugfix

```diff
@@ -0,0 +1 @@
+Fix a deprecation warning
```

Comment:
@ale-rt maybe we can be a bit more specific here :)

Reply:
Is it fine now?